ketcher-autotests/tests/Macromolecule-editor/Polymer-Bond-Tool/snake-bond-tool.spec.ts

```diff
@@ -0,0 +1,253 @@
+import { test } from '@playwright/test';
+import {
+  addMonomerToCanvas,
+  selectSingleBondTool,
+  selectSnakeBondTool,
+  takePageScreenshot,
+  waitForPageInit,
+} from '@utils';
+import { turnOnMacromoleculesEditor } from '@utils/macromolecules';
+import { bondTwoMonomers } from '@utils/macromolecules/polymerBond';
+/* eslint-disable no-magic-numbers */
+
+const MONOMER_NAME_TZA = 'Tza___3-thiazolylalanine';
+const MONOMER_ALIAS_TZA = 'Tza';
+
+test.describe('Snake Bond Tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+    await turnOnMacromoleculesEditor(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takePageScreenshot(page);
+  });
+
+  test('Create snake bond between peptides', async ({ page }) => {
+    /* 
+    Test case: #3280 - Create snake bond 
+    Description: Snake bond tool
+    */
+
+    await selectSnakeBondTool(page);
+
+    const peptide1 = await addMonomerToCanvas(
+      page,
+      MONOMER_NAME_TZA,
+      MONOMER_ALIAS_TZA,
+      300,
+      300,
+      0,
+    );
+    const peptide2 = await addMonomerToCanvas(
+      page,
+      MONOMER_NAME_TZA,
+      MONOMER_ALIAS_TZA,
+      400,
+      400,
+      1,
+    );
+    const peptide3 = await addMonomerToCanvas(
+      page,
+      MONOMER_NAME_TZA,
+      MONOMER_ALIAS_TZA,
+      300,
+      500,
+      2,
+    );
+    const peptide4 = await addMonomerToCanvas(
+      page,
+      MONOMER_NAME_TZA,
+      MONOMER_ALIAS_TZA,
+      200,
+      200,
+      3,
+    );
+
+    await selectSingleBondTool(page);
+
+    await bondTwoMonomers(page, peptide1, peptide2);
+    await bondTwoMonomers(page, peptide2, peptide3);
+    await bondTwoMonomers(page, peptide3, peptide4);
+  });
+
+  test.skip('Check snake mode arrange', async ({ page }) => {
```

Comment:
Do we need to skip it?

Reply:
Now yes, because I am waiting changes from new master - it doesn't work without it, but soon I will remove it)